app/src/main/java/com/x8bit/bitwarden/ui/platform/feature/vaultunlockednavbar/VaultUnlockedNavBarViewModel.kt

```diff
@@ -17,22 +22,19 @@ class VaultUnlockedNavBarViewModel @Inject constructor(
     private val authRepository: AuthRepository,
     specialCircumstancesManager: SpecialCircumstanceManager,
 ) : BaseViewModel<VaultUnlockedNavBarState, VaultUnlockedNavBarEvent, VaultUnlockedNavBarAction>(
-    initialState = run {
-        val hasOrganization = authRepository
-            .userStateFlow
-            .value
-            ?.activeAccount
-            ?.organizations
-            ?.isNotEmpty()
-            ?: false
-        val vaultRes = if (hasOrganization) R.string.vaults else R.string.my_vault
-        VaultUnlockedNavBarState(
-            vaultNavBarLabelRes = vaultRes,
-            vaultNavBarContentDescriptionRes = vaultRes,
-        )
-    },
+    initialState = VaultUnlockedNavBarState(
```

Comment:
Hmmm, so this is not guaranteed to be correct on the first go but the event we listen too below will update it?

Reply:
Correct. I'm not sure why it's not correct on init - I guess it just takes a moment to load the active account and determine if it has organizations? In this update version, I noticed that the tab bar title and the title of the screen itself update at the exact same time, which makes sense if that's the case

Reply:
So just to add a bit to this, it looks like there may be a "race" with the organizations list of the active user after login(?). Going through the previous code in the debugger the orgs list is empty, then if I lock the same account and unlock the count will be 1 (for lukus+policy). Which is why I think this one works cause we are observing the state change in the user state value when the combine in `AuthRepostitoryImpl` emits the org list.